integration_test/support/models.exs

```diff
@@ -171,5 +172,38 @@ defmodule Ecto.Integration.Tag do
   schema "tags" do
     field :ints, {:array, :integer}
     field :uuids, {:array, Ecto.UUID}
+    embeds_many :items, Ecto.Integration.Item
+  end
+end
+
+defmodule Ecto.Integration.Item do
+  @moduledoc """
+  This module is used to test:
+
+    * Embedding
+
+  """
+  use Ecto.Integration.Model
+
+  # TODO decide what to do with source here
```

Comment:
Let's remove this TODO and keep this model as is. Let's discuss adding an embed shortcut in the future although we shouldn't change this code to use (so we continue testing PKs are not included).